pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   flutter: ">=1.24.0-6.0.pre"
 
 dependencies:
-  analyzer: ^2.1.0
+  analyzer: ^3.0.0
```

Comment:
Maybe I could update all dependencies. I've made this PR because it is conflicting with latest version of other packages, mainly freezed :)

Sarajevo :D. Ti?

Reply:
Will merge it and update everything.

Sarajevo, ali trenutno u Berlinu